docker-compose.yml

```diff
@@ -21,13 +21,19 @@ services:
        context: .
        dockerfile: Dockerfile
        args:
-         woocommerce_version: 3.1.0
+         woocommerce_version: 4.0.1
      ports:
        - "8000:80"
      restart: always
      environment:
+       WORDPRESS_DEBUG: 1
        WORDPRESS_DB_HOST: db:3306
        WORDPRESS_DB_PASSWORD: wordpress
+       WORDPRESS_CONFIG_EXTRA: |
+        define( 'WPLANG', '${WPLANG}');
+        define( 'WP_DEBUG_LOG', '/tmp/wp-errors.log');
```

Comment:
Because we're mounting into the `wp-content` directory the WP user doesn't have permissions to write to the default log location, so we need to point to somewhere it has permissions to write to. I picked `/tmp` for no particular reason, if there's another preferred location let me know 😄